pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -638,10 +649,26 @@ private void convertInfoToJSONAndExportToES(Handle handle,
                     .doc(value, XContentType.JSON)
                     .docAsUpsert(true);
             bulkRequest.add(updateRequest);
+
+            if (isOsteoParticipantStructuredIndex) {
+                UpdateRequest updateRequestOS2 = new UpdateRequest()
+                        .index(CMI_OSTEO2_INDEX)
+                        .id(key)
+                        .doc(value, XContentType.JSON)
+                        .docAsUpsert(true);
+                finalBulkRequestOS2.add(updateRequestOS2);
+            }
         });
 
-        BulkResponse bulkResponse = esClient.bulk(bulkRequest, RequestOptions.DEFAULT);
+        if (finalBulkRequestOS2 != null) {
+            log.info("exporting {} participant records to index participants_structured.cmi.cmi-osteo2", participantRecords.size());
+            BulkResponse bulkResponse2 = esClient.bulk(finalBulkRequestOS2, RequestOptions.DEFAULT);
+            if (bulkResponse2.hasFailures()) {
+                log.error(bulkResponse2.buildFailureMessage());
+            }
+        }
```

Comment:
Q:

1. Is it intentional to update OS2 index before OS1 index?
2. If OS2 update fails, should run OS1 update always or stops?

Reply:
we are just logging and continuing. 
DSM wanted to use OS2 , new index as new osteo2 index so just made the call to do it first.